src/main/webapp/app/exercises/shared/result/result-detail.component.html

```diff
@@ -56,16 +58,26 @@ <h4 class="modal-title">Feedback</h4>
                 </div>
                 <div *ngFor="let feedback of filteredFeedbackList" [ngClass]="['feedback-item', 'alert', getClassNameForFeedbackItem(feedback)]" id="feedback-message">
                     <span>{{ feedback.category }}</span>
-                    <span id="feedback-points" class="feedback-points" *ngIf="feedback.credits"
-                        >{{ roundScoreSpecifiedByCourseSettings(feedback.credits, course) }}P<fa-icon
+                    <span id="feedback-points" class="feedback-points" *ngIf="feedback.credits">
+                        <ng-container *ngIf="feedback.actualCredits && feedback.actualCredits != feedback.credits">
+                            {{ roundScoreSpecifiedByCourseSettings(feedback.actualCredits, course) }}P/
+                        </ng-container>
+                        {{ roundScoreSpecifiedByCourseSettings(feedback.credits, course) }}P
+                        <fa-icon
                             *ngIf="feedback.type === FeedbackItemType.Subsequent"
                             [icon]="faExclamationTriangle"
                             [ngbTooltip]="'artemisApp.assessment.subsequentFeedback' | artemisTranslate"
                         ></fa-icon>
+                        <fa-icon
+                            *ngIf="feedback.type === FeedbackItemType.Issue && feedback.actualCredits && feedback.actualCredits != feedback.credits"
```

Comment:
The second check (`feedback.actualCredits`) would evaluate to false for actualCredits == 0 (since 0 is falsy). I feel like this should be not the case here.
```suggestion
                            *ngIf="feedback.type === FeedbackItemType.Issue && feedback.actualCredits != null && feedback.actualCredits !== feedback.credits"
```